apps/notifications/src/modules/alert/services/chain-message-alert/chain-message-alert.service.spec.ts

```
@@ -173,18 +179,30 @@ describe(ChainMessageAlertService.name, () => {
+          alert: {
+            prev: expect.objectContaining({ id: alert2.id }),
+            next: expect.objectContaining({ id: alert2.id })
+          },
+          data: event
+        }
       });
       expect(onMessage).toHaveBeenCalledWith(alertMessage2);
     });
 
-    it("should correctly handle complex condition matching", async () => {
+    fit("should correctly handle complex condition matching", async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)